doc/nrf/gs_programming.rst

```diff
@@ -87,6 +87,31 @@ Complete the following steps to build |NCS| projects with SES after :ref:`instal
    .. build_SES_projimport_start
 
 4. Click :guilabel:`OK` to add the project to SES.
+
+License activation for first-time use
+=====================================
```

Comment:
nit: Doesn't this header suggest the license needs to be activated also for later uses?
```suggestion
License activation
==================
```
I think we don't need to repeat "first time" if we state it in the first sentence.

Reply:
Also, this is a static header, but we're requiring action from the user, so I would go for something like "Activating license".

Reply:
After an offline discussion with greg - will update this as - Activating license for first time (As the user needs to do the license activation just for the 1st time).